allure-web/src/layouts/testcase/TestcaseLayout.js

```diff
@@ -0,0 +1,27 @@
+import PaneLayout from '../pane/PaneLayout';
+import BackPanelView from '../../components/back-panel/BackPanelView';
+import router from '../../router';
+
+export default class TestcaseLayout extends PaneLayout {
+    loadData() {
+        return Promise.resolve();
+    }
+
+    onStateChange() {
+        const changed = Object.assign({}, this.state.changed);
+        const paneView = this.getChildView('content');
+        if(router.previousUrl && !router.previousUrl.startsWith('testcase') && !paneView.getRegion('back')) {
```

Comment:
This workaround with `startsWith` is terrible.

Maybe we should consider an approach wth query param. We can write a helper `{{testcase-link uid=uid}}` that will render a link to a test case based on current url.